persistence-elasticsearch/core/src/main/java/org/apache/unomi/persistence/elasticsearch/ElasticSearchPersistenceServiceImpl.java

```diff
@@ -1158,6 +1157,45 @@ protected Boolean execute(Object... args) throws Exception {
         }
     }
 
+    private void startTaskTracker(AbstractBulkByScrollRequest request, TaskSubmissionResponse response) {
```

Comment:
May I suggest to rename this method to waitForTask ? With the name start I was expecting to see another system to wait for the ending or for polling for completion.

Reply:
Thx, it's renamed.
At first it was a separate function that's why.